config/filter.d/postfix.conf

```diff
@@ -52,6 +52,10 @@ mdre-aggressive = %(mdre-auth2)s
 mdpr-errors = too many errors after \S+
 mdre-errors = ^from [^[]*\[<HOST>\]%(_port)s$
 
+# Extra mode "screendnsbl", triggered on postfix/postscreen[<PID>]: DNSBL rank <NUM> for [<HOST>]:<PORT>
```

Comment:
I guess it is wrong mode name for DNS-based Blackhole List, correct would be `dnsbl`, but much better why not combine it with `rbl`?

Reply:
intention was to implement an optional feature, not extending/overwriting existing definitions leading to by others unexpected behavior